Upgrade 12.1.6.Final (#6)
infinispan/infinispan-server-tutorial: README.adoc

To complete this tutorial, you need:

- Approximately 25 to 30 minutes

> - IDE such as Eclipse, Intellij

Reviewer: ```suggestion
- IDE such as Eclipse or Intellij
```

==== Create a Temperature Cache

> Update the `connect()` method in the `DataSourceConnector` class add a `remoteCache("temperature")` as follows:

Reviewer: ```suggestion
Update the `connect()` method in the `DataSourceConnector` class by adding a `remoteCache("temperature")` as follows:
```

.org.infinispan.tutorial.db.DataSourceConnector
[source,java]
----

> builder.remoteCache("temperature").configurationURI(temperatureCacheConfig);

Reviewer: ```suggestion
builder.remoteCache("temperature").configurationURI(temperatureCacheConfig); <1>
```

In this step, you create a `weather` cache that can store `LocationWeather` objects. First you must initialize the marshalling context in the application and then create the cache, as follows:

> As you did before, configure the 'weather' cache.
> 
> .org.infinispan.tutorial.db.DataSourceConnector
> [source,java]
> ----
> builder.remoteCache("weather").configurationURI(weatherCacheConfig);
> ----
> <1> Adds a cache named `weather` that uses the content of the 'weatherCacheConfig.xml' file.
> +
> 
> Unlike the `temperature` cache, `weather` cache will store complex java objects and queries will
> be performed on the values. This is why the serialization context needs to be registered on the client
> and on the Infinispan Server.

Reviewer: ```suggestion
As before, configure the `weather` cache.

.org.infinispan.tutorial.db.DataSourceConnector
[source,java]
----
builder.remoteCache("weather").configurationURI(weatherCacheConfig); <1>
----
<1> Adds a cache named `weather` that uses the content of the 'weatherCacheConfig.xml' file.

Unlike the `temperature` cache, the `weather` cache stores complex Java objects and you will query 
the values. For this reason the serialization context needs to be registered on the client
and on Infinispan Server.
```
